mypy/server/aststrip.py

```diff
@@ -180,49 +180,44 @@ def process_lvalue_in_method(self, lvalue: Node) -> None:
                 self.process_lvalue_in_method(item)
 
     def visit_import_from(self, node: ImportFrom) -> None:
-        if node.assignments:
-            node.assignments = []
-        else:
-            # If the node is unreachable, don't reset entries: they point to something else!
-            if node.is_unreachable: return
-            if self.names:
-                # Reset entries in the symbol table. This is necessary since
-                # otherwise the semantic analyzer will think that the import
-                # assigns to an existing name instead of defining a new one.
-                for name, as_name in node.names:
-                    imported_name = as_name or name
-                    # This assert is safe since we check for self.names above.
-                    assert self.file_node is not None
-                    sym = create_indirect_imported_name(self.file_node,
-                                                        node.id,
-                                                        node.relative,
-                                                        name)
-                    if sym:
-                        self.names[imported_name] = sym
+        node.assignments = []
+
+        # If the node is unreachable, don't reset entries: they point to something else!
+        if node.is_unreachable: return
+        if self.names:
+            # Reset entries in the symbol table. This is necessary since
+            # otherwise the semantic analyzer will think that the import
+            # assigns to an existing name instead of defining a new one.
+            for name, as_name in node.names:
+                imported_name = as_name or name
+                # This assert is safe since we check for self.names above.
+                assert self.file_node is not None
+                sym = create_indirect_imported_name(self.file_node,
+                                                    node.id,
+                                                    node.relative,
+                                                    name)
+                if sym:
+                    self.names[imported_name] = sym
 
     def visit_import(self, node: Import) -> None:
```

Comment:
I see logically three changes, but only two tests added. Maybe there is missing a test with a simple import? (i.e. the one visited by this method)

Reply:
The change was for consistency, but it turns out that `Import` can't even have `assignments`. Changing it to an assert instead.